internal/transport/http2_client.go

```diff
@@ -526,7 +526,7 @@ func (t *http2Client) getPeer() *peer.Peer {
 func (t *http2Client) outgoingGoAwayHandler(g *goAway) (bool, error) {
 	t.mu.Lock()
 	defer t.mu.Unlock()
-	if err := t.framer.fr.WriteGoAway(t.nextID-2, http2.ErrCodeNo, g.debugData); err != nil {
+	if err := t.framer.fr.WriteGoAway(math.MaxInt32*3/4, http2.ErrCodeNo, g.debugData); err != nil {
```

Comment:
This is not right.  We want to send the GOAWAY with the ID of the last stream started.  We need to find another way to fix the race.